network_modeling/configs/templates/1.3/dellmellanox/common/banner-motd-dell.j2

```diff
@@ -1,8 +1,8 @@
 {# blank line below required #}
 
-banner motd {{ '^' }}
-###############################################################################
-# CSM version:  {{ variables.CSM_VERSION }}
-# CANU version: {{ variables.CANU_VERSION }}
-###############################################################################
-{{ '^' }}
+banner motd {{ '^C' }}
+ ###############################################################################
+ # CSM version:  {{ variables.CSM_VERSION }}
+ # CANU version: {{ variables.CANU_VERSION }}
+ ###############################################################################
```

Comment:
Is the addition of space intentional?

Reply:
Unfortunately yes :( Dell auto adds two spaces no matter what.

Reply:
Yes, Dell is super tricky/critical of what it accepts.